doc/stdlib/arrays.rst

```diff
@@ -1198,41 +1214,121 @@ to/from the latter via ``Array(bitarray)`` and ``BitArray(array)``, respectively
 
    Performs a bitwise not operation on ``B``\ . See :ref:`~ operator <~>`\ .
 
-.. function:: rol!(dest::BitArray{1}, src::BitArray{1}, i::Integer) -> BitArray{1}
+   .. doctest::
+
+       julia> A = trues(2,2)
+       2×2 BitArray{2}:
+        true  true
+        true  true
+
+       julia> flipbits!(A)
+       2×2 BitArray{2}:
+        false  false
+        false  false
+
+.. function:: rol!(dest::BitVector, src::BitVector, i::Integer) -> BitVector
 
    .. Docstring generated from Julia source
 
-   Performs a left rotation operation on ``src`` and put the result into ``dest``\ .
+   Performs a left rotation operation on ``src`` and puts the result into ``dest``\ . ``i`` controls how far to rotate each bit.
```

Comment:
all bits are rotated by the same amount. isn't this just "how many bits to rotate" ?